server/sockets/list.js

```diff
@@ -348,6 +367,91 @@ const emitRemoveMemberOnLeaveCohort = socket =>
       });
   });
 
+const changeListType = (socket, dashboardClients, cohortClients, listClients) =>
+  socket.on(ListActionTypes.CHANGE_TYPE_SUCCESS, (data, fn) => {
```

Comment:
what is this `fn` for?

Reply:
I forgot to remove it. This is completely unnecessary. I'm sorry.